### PR TITLE
Add new --convert-umlauts parameter

### PR DESCRIPTION
**Pull request by 5p1n**
This new parameter adds functionality to convert the common Latin-1 umlauts ä, ö, ü and ß to ae, oe, ue and ss

### CODE REVIEW

**Comment by digininja** [comment]
Looks ok, have you got a site I can easily test it on?

**Comment by 5p1n** [comment]
I've been using this website for testing purposes:
https://www.studying-in-germany.org/german-umlauts/

**Comment by digininja** [comment]
That seems like one that would be pretty good for looking at umlauts. I'll
give your change a test and see, but it looks ok.

**Comment by digininja** [comment]
I've just merged the lowercase as that was easier to test but now I'm getting conflicts. If you are any good with Git, can you fix them and update this PR.

If not, I'll have a go later.

**Comment by 5p1n** [comment]
Should be good to go.

File: cewl.rb [hunk 1]
```diff
@@ -982,6 +987,11 @@ def usage
 								words.gsub!(/[^[[:alpha:]]]/i, " ")
 							end
 
+							if convert_umlauts then
+								words.gsub!(/[^a-zäöüßÄÖÜ]/i, " ")
```
Comment (digininja) [inline cewl.rb]: Why are you doing this? It will lose characters like ë, è, é, and ê

Comment (5p1n) [reply cewl.rb]: You're right - This line has to be removed. (This code is a remnant of another use case in which I had to replace the most common umlauts and remove all the remaining/other ones.)

**Comment by digininja** [comment]
Thanks.

**Pull request merged**